src/routes/index.js

```diff
@@ -12,8 +14,9 @@ export function routes() {
   const router = new Router();
   router.use(dockerFlowRoutes().routes());
 
-  // Versioning applies only to API routes, that's why we specify it here.
+  // Versioning and CORS applies only to API routes, that's why we specify it here.
   router.use(versioning(1));
+  router.use(cors());
```

Comment:
It might be worth mentioning that the CORS is using the default configuration here.